rest/slurmrestapi.py

```diff
@@ -118,20 +119,38 @@ def login():
 @cache()
 def get_jobs():
     jobs = pyslurm.job().get()
+    onlyUsersJobs = False
+    currentUser = None
+    result = {}
+    if auth_enabled:
+        # Fetch the attributs of private_data
+        config = pyslurm.config().get()
```

Comment:
The logic to extract the configuration, parse the privatedata and set a boolean should done in a factorized function since this code is very similar in the various routes.